python/ganga/__init__.py

```diff
@@ -20,8 +20,10 @@ def ganga_license():
 from Ganga.Core.InternalServices import ShutdownManager
 ShutdownManager.install()
 
-## TODO need to implement loading of the config system properly here.
-## loadPlugins and autoPopulateGPI will take this into account when loading objects
+system_vars = {}
+for opt in Ganga.Utility.Config.getConfig('System'):
+    system_vars[opt] = Ganga.Utility.Config.getConfig('System')[opt]
```

Comment:
Perhaps, but for the call to `configure()`, I thought it best to use the explicit `Ganga.Utility.Config.configure()` rather than just `configure()` to make it clear *what* is being configured.

It becomes the difference between:
```python
from Ganga.Utility.Config import getConfig, configure

system_vars = {}
for opt in getConfig('System'):
    system_vars[opt] = getConfig('System')[opt]
configure([os.path.expanduser('~/.gangarc')], system_vars)  # This is ambiguous in my view
```
and
```python
import Ganga.Utility.Config

system_vars = {}
for opt in Ganga.Utility.Config.getConfig('System'):
    system_vars[opt] = Ganga.Utility.Config.getConfig('System')[opt]
Ganga.Utility.Config.configure([os.path.expanduser('~/.gangarc')], system_vars)
```

Perhaps a compromise would be:
```python
from Ganga.Utility import Config

system_vars = {}
for opt in Config.getConfig('System'):
    system_vars[opt] = Config.getConfig('System')[opt]
Config.configure([os.path.expanduser('~/.gangarc')], system_vars)
```
or
```python
from Ganga.Utility.Config import getConfig
import Ganga.Utility.Config

system_vars = {}
for opt in getConfig('System'):
    system_vars[opt] = getConfig('System')[opt]
Ganga.Utility.Config.configure([os.path.expanduser('~/.gangarc')], system_vars)
```
What are your thoughts?